hydra-node/exe/tx-cost/TxCost.hs

```diff
@@ -85,7 +85,7 @@ computeInitCost = do
 computeCommitCost :: IO [(NumUTxO, TxSize, MemUnit, CpuUnit)]
 computeCommitCost = do
   interesting <- catMaybes <$> mapM compute [1, 2, 3, 5, 10, 50, 100]
-  limit <- maybeToList . getFirst <$> foldMapM (fmap First . compute) [100 .. 500]
+  limit <- maybeToList . getFirst <$> foldMapM (fmap First . compute) [500, 499 .. 101]
```

Comment:
Smart idea to show higher values first!

Reply:
Not sure if you mean that, but it will only show show only one result for `limit` -> the `First` to be `Just`.

Reply:
Not exactly, I did not pay attention to the `getFirst`, it's even better :)